libraries/cerner_tomcat.rb

```diff
@@ -154,6 +155,7 @@ def action_enable
               user new_resource.user
               group new_resource.group
               only_if { new_resource.create_user }
+              shell new_resource.user_shell unless new_resource.user_shell.nil?
```

Comment:
You can see the parameter in `poise_service_user` [here](https://github.com/poise/poise-service/blob/master/lib/poise_service/resources/poise_service_user.rb#L71)